scripts/src/errors.ts

```diff
@@ -14,34 +14,34 @@ const CODES = {
 		MissingToken: 1,
 		InvalidToken: 2,
 		InvalidApiKey: 3,
-		JwtKidMissing: 4,
-		TOSMissingOrOldToken: 5
+		TOSMissingOrOldToken: 5,
```

Comment:
shouldn't be 4 ?

Reply:
I didn't want to break the current number. Are you or @Elaz  using this error?

Reply:
nop, you can change